test/e2e/generate_systemd_test.go

```diff
@@ -53,6 +53,18 @@ var _ = Describe("Podman generate systemd", func() {
 		Expect(session).To(ExitWithError())
 	})
 
+	It("podman generate systemd bad restart-policy value", func() {
+		session := podmanTest.Podman([]string{"create", "--name", "foobar", "alpine", "top"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		session = podmanTest.Podman([]string{"generate", "systemd", "--restart-policy", "bogus", "foobar"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).To(ExitWithError())
+		found, _ := session.ErrorGrepString("Error: bogus is not a valid restart policy")
```

Comment:
shouldn't bogus be arg[3]?

Reply:
Do you want to change the error message to point to the exact argument?

Reply:
well, the bad value passed in? unless thats a bad idea? arg[4]?

Reply:
The passed value is part of the error message already. It's "Error: $bad-value ...".

Reply:
gotcha, I'm just bad at reading the code. I saw the string on 64 and thought it was hard coded to bogus. I see its part of the return of ExitWithError(), my bad 😊 nvm